edit idea

NielsSensei opened the pull request:
Edit Idea gefixt.
Nieuwe JS die gelijkaardig is aan closedfield.js => editclosedfield.js

Fixed shirt with Destroy & Make Ideation

NielsSensei committed:
FINALLY

Edit Idea gefixt.
Nieuwe JS die gelijkaardig is aan closedfield.js => editclosedfield.js

Fixed shit with Destroy & Make Ideation

XanderVeldeman commented:
ik heb een nieuwe comment gemaakt (op een nieuw account) maar opsommingen toevoegen bij edit werkt niet 
![image](https://user-images.githubusercontent.com/25788097/57987192-76da5980-7a7e-11e9-98aa-84612f0deed4.png)

XanderVeldeman reviewed (changes requested):
zie comment ;)

NielsSensei committed:
Doing some Fixing

Deftig opslaan van ClosedField

Per idee 1 ideefields row

Ook een link verificatie als je een link toevoegt bij maken van ideation & videofield & aanpassen ideation.

Iets gefixt met ideation aanpassing. hij probeert soms hem te scheiden van zijn ouders. Iets wat alleen Theo Francken mag doen.

XanderVeldeman reviewed (approved):
Ite, it works now, go ahead!
